core/domain/collection_domain.py

```diff
@@ -906,133 +729,69 @@ def exploration_ids(self):
         return [node.exploration_id for node in self.nodes]
 
     @property
-    def init_exploration_ids(self):
-        """Returns a list of exploration IDs that are starting points for this
-        collection (ie, they require no prior skills to complete). The order
-        of these IDs is given by the order each respective exploration was
-        added to the collection.
+    def init_exploration_id(self):
```

Comment:
Let's call this first_exploration_id; init() is sometimes short for initialize().

Reply:
Done.